src/tools/genpolicy/rules.rego

```diff
@@ -1580,9 +1581,8 @@ allow_interactive_exec(p_container, i_process) {
     print("allow_interactive_exec: true")
 }
 
-# TODO: should other ExecProcessRequest input data fields be validated as well?
 ExecProcessRequest {
-    print("ExecProcessRequest 1: input =", input)
+    allow_exec_process_input(input)
```

Comment:
Keep the log so we know which ExecProcessRequest body we are in (1 , 2, or 3) But the `input` only gets printed now in allow_exec_process_input
```suggestion
     print("ExecProcessRequest 1: start")
    allow_exec_process_input(input)
```

Reply:
Then we should be able to remove the first print from the new allow_exec_process_input